TOC.md

```diff
@@ -177,7 +177,7 @@ shows only GitHub based contributions and does not capture all the contributions
 we value. **We expect this metric not to capture everyone who should be eligible
 to vote.** If a community member has had significant contributions over the past
 year but is not captured in the lfanalytics.io dashboard, they will be able
-to submit an exception form to the steering committee who will then review and
+to submit an exception form to the current TOC who will then review and
```

Comment:
Should we explicitly allow an escalation to the Governing Board?

Reply:
Added